Holovibes/cuda_includes/bw_area_filter.cuh

```diff
@@ -0,0 +1,67 @@
+/*! \file
+ *
+ * \brief function use for black and white area filter algo
+ */
+#pragma once
+
+#include "frame_desc.hh"
+
+/**
+ * \brief write a matrix with connected component (labels_d) and store the size of each component in an other matrix
+ * (label_sizes_d), this function use TwoPass algorithm, and does not handle the border of the image
+ *
+ * \param[in] image_d The image to process (GPU Memory)
+ * \param[out] labels_d  The matrix use to store label of each pixel (GPU Memory)
+ * \param[out] labels_sizes_d the matrix use to store size of each label (GPU Memory)
+ * \param[out] linked_d the matrix use to store linked of each label to an other label (GPU Memory), no need to use it
+ * after this function
+ * \param[out] size_t_gpu_ Size_t on GPU Memory use to store a mutex
+ * \param[in] width Width of the frame
+ * \param[in] height Height of the frame
+ * \param[in] stream The CUDA stream on which to launch the operation
+ */
+void get_connected_component(const float* image_d,
+                             size_t* labels_d,
+                             size_t* labels_sizes_d,
+                             size_t* linked_d,
```

Comment:
Maybe use a uint instead since the max value will be frame_res.
Also outputs buffer should be before inputs buffer